jitbuilder/release/Makefile

```diff
@@ -44,6 +45,7 @@ ALL_TESTS = \
             toiltype
 
 goal: $(ALL_TESTS)
+	./transactionaloperations
```

Comment:
Make sure that the behaviour you describe is documented somewhere on the service to make it clear as it could be surprising to some.